packages/toggleBox/components/ToggleBox.tsx

```diff
@@ -1,5 +1,4 @@
 import * as React from "react";
-import shortid from "shortid";
```

Comment:
Why are we removing this?

Reply:
Because ID is actually required for this component, so this was unnecessary